adapters/base/build.gradle

```diff
@@ -59,7 +60,7 @@ publishing {
         maven(MavenPublication) {
             from components.java
             groupId project.group
-            artifactId "${archivesBaseName}"
+            artifactId "adapters-base"
```

Comment:
how about `base-adapter`?

Reply:
`adapters-base` matches the directory structure `adapters/base` but I suppose `base-adapter` rolls off the tongue a bit more easily - @yifan-c any opinions here (I'll leave it as is until we come to consensus but don't have a strong opinion either way)

Reply:
Just providing a data point
https://mvnrepository.com/search?q=base
All the artifacts in the first page has `-base` instead of `base-`

Reply:
in that case I +1 -base :) . Thanks for the data point